pyqir-evaluator/tests/test_unknown_external_func.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from pyqir.evaluator import GateLogger, NonadaptiveEvaluator
+import pytest
+import tempfile
+
+
```

Comment:
This is looking great. I think we need a test case for use of multiple non-supported calls.

Reply:
Done in d9741dd